yarn.lock

```diff
@@ -2,17 +2,292 @@
 # yarn lockfile v1
 
 
+"@mrmlnc/readdir-enhanced@^2.2.1":
```

Comment:
Note that the Yarn lock files are woefully out of date in every Readium JS projects ... we should really remove them as only the NPM lock files are maintained.